controller/workitemtypes_blackbox_test.go

```diff
@@ -67,9 +67,7 @@ func (s *workItemTypesSuite) TestList() {
 
 	s.T().Run("ok", func(t *testing.T) {
 		// when
-		// Paging in the format <start>,<limit>"
-		page := "0,-1"
-		res, witCollection := test.ListWorkitemtypesOK(t, nil, nil, s.typeCtrl, fxt.Spaces[0].ID, &page, nil, nil)
+		res, witCollection := test.ListWorkitemtypesOK(t, nil, nil, s.typeCtrl, fxt.Spaces[0].ID, nil, nil)
```

Comment:
How about using golden files here & below test cases?

Reply:
@baijum I've added golden files in 32cd737c61da126f4e9c28ed58ed5c46545ae15f